org.jacoco.core.test/src/org/jacoco/core/analysis/AnalyzerTest.java

```diff
@@ -90,8 +91,10 @@ public void should_ignore_synthetic_classes() throws Exception {
 	}
 
 	@Test
-	public void should_analyze_java10_class() throws Exception {
-		final byte[] bytes = createClass(BytecodeVersion.V10);
+	public void should_analyze_class_greater_than_max_version()
```

Comment:
While "downgrade" trick can be removed, this test must be preserved under another name to prevent regressions, because it verifies that `Analyzer` computes `classId` using original class bytes.